src/common/components/Header/HeaderScreen.tsx

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import { withRouter, useHistory } from 'react-router'
+import { Link } from 'react-router-dom'
+import styles from './HeaderScreen.module.css'
+import routes from '../../constants/routes.json'
+import Logo from '../../assets/icons/ico-logo.svg'
+import addBtn from '../../assets/icons/ico-addbtn.png'
+import searchIcon from '../../assets/icons/ico-search.svg'
+import QuestionTag from '../../assets/icons/ico-question.svg'
+import BellIcon from '../../assets/icons/ico-bell.svg'
+import MessageIcon from '../../assets/icons/ico-msg.svg'
+import SettingIcon from '../../assets/icons/ico-setting.svg'
+import UserIcon from '../../assets/icons/ico-user.svg'
+import cn from 'classnames'
+
+import Icon from '../Icon'
+
+interface IMenuType {
```

Comment:
Please use the Type instead of Interface
interface IMenuType would be Type TIMenu

Reply:
fixed